docs/sources/enterprise/license-restrictions.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "License restrictions"
+description = "Grafana Enterprise license restrictions"
+keywords = ["grafana", "licensing", "enterprise"]
+weight = 8
++++
+
+# License restrictions
+
+Enterprise licenses are limited by the number of active users, a license expiration date, and the URL of the Grafana instance.
+
+**User limits**
+
+Grafana licenses allow a certain number of active users in an instance. An active user is any user that has signed in to Grafana within the past 30 days.
+
+In the context of licensing, each user is classified as either a viewer or an editor, which are defined as follows:
+
+- An editor is a user who has the ability to edit and save dashboards, which includes:
+    - Grafana server admins
+    - Users assigned an Org role of "Editor" or "Admin," 
+    - Users that have been granted "Admin" or "Edit" permissions through [dashboard/folder permissions](https://grafana.com/docs/grafana/latest/permissions/dashboard_folder_permissions/).     
```

Comment:
```suggestion
    - Users who have been granted `Admin` or `Edit` permissions. See [Dashboard and folder permissions](https://grafana.com/docs/grafana/latest/permissions/dashboard_folder_permissions/).
```

Reply:
Can a user be granted permissions another way? Line 21 seems quite dependent on the _how_. Does my change make this statement less accurate?

Reply:
Hm, I'm a bit hesitant about the suggested change. The above sentence in line 20 already talks about `Admin` and `Edit` roles in organisation level, so the intention of this sentence was to stress that the same applies if you give `Admin` or `Edit` permissions on dashboard/folder level and I thought referencing the docs will make it even more explicit. Maybe `through` is not the best word to use to stress that out, but I hope I was able to explain what I meant to say there.

Reply:
Yes, "through" is off here. Hmm, how about ...

```suggestion
    - Users that have been granted "Admin" or "Edit" permissions at the dashboard or folder level. Refer to [Dashboard and folder permissions](https://grafana.com/docs/grafana/latest/permissions/dashboard_folder_permissions/).     
```

Reply:
Nice thanks! Applied the suggestion!